### ISSUE TITLE
Failed to precompile with Julia 1.10-beta1

### ISSUE THREAD

**Issue by PhyX-Meow** [brianguenter/FastDifferentiation.jl#31]
```
In [1]: using FastDifferentiation
Precompiling FastDifferentiation
  ✗ FastDifferentiation
  0 dependencies successfully precompiled in 7 seconds. 92 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

FastDifferentiation [eb9bf01b-bf85-4b60-bf87-ee5de06c00be]

Failed to precompile FastDifferentiation [eb9bf01b-bf85-4b60-bf87-ee5de06c00be] to "/home/wuli/.julia/compiled/v1.10/FastDifferentiation/jl_ilJS7X".
WARNING: Method definition transpose(FastDifferentiation.Node{T, N} where N where T) in module FastDifferentiation at /home/wuli/.julia/packages/SymbolicUtils/NJ0fs/src/methods.jl:83 overwritten at /home/wuli/.julia/packages/FastDifferentiation/oGU1A/src/ExpressionGraph.jl:291.
ERROR: LoadError: Method overwriting is not permitted during Module precompile.
```

**Comment by brianguenter** [comment]
Are you using 1.10? Method overwriting is a warning in 1.9 but an error in 1.10. I'm in the process of updating the code so this doesn't happen.

**Comment by brianguenter** [comment]
Fixed in v 0.2.11 which should be available on the Julia registry in a few minutes.
